src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -86,6 +121,14 @@ public HubConnection(IConnectionFactory connectionFactory, IHubProtocol protocol
             }
         }
 
+        /// <summary>
+        /// Subscribes a handler that will be raised when a hub method with the specified method and parameters is invoked.
+        /// </summary>
+        /// <param name="methodName">The name of the hub method to subscribe to.</param>
+        /// <param name="parameterTypes">The parameter types of the hub method to subscribe to.</param>
+        /// <param name="handler">The handler that will be raised when a matching hub method is invoked.</param>
```

Comment:
`The action that will be performed when the hub method is invoked`

Source: https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.taskawaiter.oncompleted?view=netstandard-2.0#System_Runtime_CompilerServices_TaskAwaiter_OnCompleted_System_Action_

(Possibly a slightly sketchy source, but I think it works well)

Reply:
We call it a handler earlier and the parameter is called handler so I prefer that over action. Also events tend to get raised.

Reply:
"handler" is fine. See above about `raised`, I'm not super worried but if we do change it, it should be `invoked` here (I wasn't even consistent in my suggestion ;))